app/controllers/data_requests_controller.rb

```diff
@@ -0,0 +1,71 @@
+class DataRequestsController < ApplicationController
+  def request_extracts
+    make_request(DataRequest.requested_data[:extracts])
+  end
+
+  def request_reductions
+    make_request(DataRequest.requested_data[:reductions])
+  end
+
+  def check_status
+    request = DataRequest.find(params[:request_id])
+    return head 404 if request.nil?
+
+    if request.pending?
+      return head 201
+    end
+
+    if request.processing?
+      return head 202
+    end
+
+    if request.failed?
+      return head 500
```

Comment:
Not sure this is a good idea. How would an API client be able to distinguish between an error while checking the status, and a failed data export?